torch/jit/_monkeytype_config.py

```diff
@@ -88,30 +106,17 @@ def consolidate_types(self, qualified_name: str) -> Dict:
             # then consolidate the type to `Any` and replace the entry
             # by type `Any`.
             for arg, types in all_args.items():
-                _all_type = " "
-                for _type in types:
-                    # If the type is a type imported from typing
-                    # like Tuple, List, Dict then replace "typing."
-                    # with a null string.
-                    if inspect.getmodule(_type) == typing:
-                        _type_to_string = str(_type)
-                        _all_type += _type_to_string.replace('typing.', '') + ','
-                    elif _type is torch.nn.parameter.Parameter:
-                        # Check if the type is torch.nn.parameter.Parameter,
-                        # use the entire quaalified name `torch.nn.parameter.Parameter`
-                        # for type
-                        _all_type += 'torch.nn.parameter.Parameter' + ','
-                    else:
-                        _all_type += _type.__name__ + ','
-                _all_type = _all_type.lstrip(" ")  # Remove any trailing spaces
-
-                if len(types) == 2 and 'NoneType' in _all_type:
+                all_type = " "
+                for type in types:
+                    all_type += get_type(type)
```

Comment:
I think we had this discussion before, but could you remind me what the reason was to not process types on type objects? Post processing type strings seems less robust.